configs/arco.json

```diff
@@ -1,25 +1,43 @@
 {
   "index_name": "arco",
   "start_urls": [
-    "https://arco.design/",
-    "https://arco.design/react/docs/start"
-  ],
-  "sitemap_urls": [
-    "https://arco.design/sitemap.xml"
+    {
+      "url": "https://arco.design/vue/*",
+      "tags": ["vue"],
+      "selectors_key": "vue"
+    },
+    {
+      "url": "https://arco.design/react/*",
+      "tags": ["react"]
+    },
+    {
+      "url": "https://arco.design.+?(en-US)",
+      "tags": ["en-US"]
+    }
```

Comment:
Matching patterns also work for `start_urls` so you don't need wildcards or variables when there's only one option

Reply:
I have changed it, but in the case of en-US, we still need to use regular to match